lib/oanda_currency.rb

```diff
@@ -1,2 +1,4 @@
 require 'money/bank/oanda_currency'
 require 'money/rates_store/rate_removal_support'
+
+require 'pry-byebug'
```

Comment:
@zuccamia Don't think we want to include pry in production. Could we remove this one?

Reply:
Yes, sorry this should be removed. Done in 2dd6fbd